src/web/mime/request.md

```diff
@@ -3,38 +3,44 @@
 [![reqwest-badge]][reqwest] [![mime-badge]][mime] [![cat-net-badge]][cat-net] [![cat-encoding-badge]][cat-encoding]
 
 When receiving a HTTP reponse from *reqwest* the [MIME type] or media type may be

```

Comment:
I made a mistake when I merged the crate names Italicized.  Can you replace *reqwest* with `reqwest` and *mime* with `mime` in this PR?